<ui version="4.0" >
 <class>ContactSettingsClass</class>
 <widget class="QWidget" name="ContactSettingsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>ContactSettings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>1</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" >
      <property name="margin" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QCheckBox" name="xstatusBox" >
          <property name="text" >
           <string>Show contact xStatus icon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="birthBox" >
          <property name="text" >
           <string>Show birthday/happy icon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="authBox" >
          <property name="text" >
           <string>Show not authorized icon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="visBox" >
          <property name="text" >
           <string>Show "visible" icon if contact in visible list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="invisBox" >
          <property name="text" >
           <string>Show "invisible" icon if contact in invisible list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="ignoreBox" >
          <property name="text" >
           <string>Show "ignore" icon if contact in ignore list</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="xstatusTextBox" >
          <property name="text" >
           <string>Show contact's xStatus text in contact list</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
